config/fam_client_interface_config.yaml

```diff
@@ -0,0 +1,42 @@
+#FAM Client Interface Service Config file
+
+# Memory Server Interface used by OpenFAM. default is direct. Other option is rpc.
+# memsrv_interface_type value of rpc implies communication between memory server and client interface via rpc.
+# memsrv_interface_type value of direct implies communication happens via function calls.
+memsrv_interface_type: direct
+
+# Metadata Server Interface used by OpenFAM. default is direct. Other option is rpc.
+# metadata_interface_type value of rpc implies communication between metadata server and client interface via rpc.
+# metadata_interface_type value of direct implies communication happens via function calls.
+metadata_interface_type: direct
+
+# List of memory servers.
+# Applicable only if memsrv_interface_type is rpc.
+#
+#Format:
+#memsrv_list :
+#- <memsrv_id1>:<host_address_1>:<port_no>
+#- <memsrv_id2>:<host_address_2>:<port_no>
+#- <memsrv_id3>:<host_address_3>:<port_no>
+#- <memsrv_id4>:<host_address_4>:<port_no>
+# Here memsrv_id is a unique number associated a with memory server.
+# And host_address can be hostname (fully qualified domain name)/ipv4 address.
+#
+# Example:
+# 5:127.0.0.1:8787
+memsrv_list:
+- 0:127.0.0.1:8787
+
+# List of metadata servers
```

Comment:
As per the new design, there should be only one metadata service and not a list of meta-data servers. similar to client interface service.